packages/editor/src/plugins/ListsPlugin/ListsPlugin.tsx

```diff
@@ -357,79 +364,101 @@ export class ListsPlugin implements IPlugin {
   private isList = (editor: Editor, node: CustomElement) => {
     return Boolean(
       Editor.isBlock(editor, node) &&
-        this.options.listTypes?.includes(node.type),
+        this.options.listTypes?.includes(node.type)
     );
   };
 
   private isListItem = (editor: Editor, node: CustomElement) => {
     return Boolean(
       Editor.isBlock(editor, node) &&
-        this.options.listItemTypes?.includes(node.type),
+        this.options.listItemTypes?.includes(node.type)
     );
   };
 
   static toggleList = (
     editor: Editor,
     listType: string,
     listItemType = ListsPlugin.LIST_ITEM_ELEMENT,
+    defaultType = "paragraph"
   ) => {
+    function isList(node: CustomElement) {
```

Comment:
I don't think we need this function, because we have static `isList` function inside of a class